apps/fxlab/app/src/main/java/com/mobileer/androidfxlab/MainActivity.kt

```diff
@@ -98,22 +98,23 @@ class MainActivity : AppCompatActivity() {
         if (Build.VERSION.SDK_INT > Build.VERSION_CODES.M) {
             handleMidiDevices()
         }
-    }
-    override fun onResume() {
-        super.onResume()
+
+        // Startup Engine and FX UI
```

Comment:
The audio stream should be opened and started in onResume()
and stopped and closed in onPause().
Otherwise it can hold onto audio resources in the background.

This is a UI Issue and the UI code should be modified.
Maybe the controller state needs to be preserved somewhere during these transitions.

Or maybe createAudioEngine needs to be broken down into an effects model part and an audio stream part.

Reply:
Done.